packages/truffle-core/lib/commands/console.js

```diff
@@ -2,6 +2,18 @@ var command = {
   command: 'console',
   description: 'Run a console with contract abstractions and commands available',
   builder: {},
+  userHelp: {
```

Comment:
How do you feel about just calling this `help` instead of `userHelp`?

Reply:
Sure, I originally did this because there was a conflict (there is already a property called 'help' in  `compile.js` for example) but now it looks like this work makes the old property obsolete.  Changing this sounds good to me.